yarn.lock

```diff
@@ -8292,6 +8292,11 @@ ci-info@^2.0.0:
   resolved "https://registry.yarnpkg.com/ci-info/-/ci-info-2.0.0.tgz#67a9e964be31a51e15e5010d58e6f12834002f46"
   integrity sha512-5tK7EtrZ0N+OLFMthtqOj4fI2Jeb88C4CAZPu25LDVUgXJ0A3Js4PMGqrn0JU1W0Mh1/Z8wZzYPxqUrXeBboCQ==
 
+ci-info@^3.2.0:
```

Comment:
This is strange, can you please run `npx yarn-deduplicate --packages=ci-info`?

Reply:
It's duplicated because the version he added in package.json is different from the version that's already present

Reply:
It's duplicated because the version `3.3.0` is installed due to the caret on `^3.2.0` on `bundle-size` package.json

Reply:
I'll dedupe it manually to force v3.2.0 on lock file